### ISSUE TITLE
I want to contribute

### ISSUE THREAD

**Issue by arunsri7** [bancodobrasil/stop-analyzing-api#30]
Hey, I am quite a newbie to programming and this is the first time I am trying to contribute to an open-source project. I have experience with working on operational tasks for 6 months in a cloud telephony company.

**Comment by tiagostutz** [comment]
Hi there @arunsri7 ! 
Nice to have you here. Do you have any experience with Golang? Are you interested in learning this language?

**Comment by arunsri7** [comment]
Yes, I do have experience in Golang. I've worked with crud functionalities, bug fixes, small feature changes, and dockerizing go apps. Also yes, I would like to learn the language.

**Comment by tiagostutz** [comment]
Awesome! ⭐ 

We are in the moment of getting the data structured in the API contract and returning that data from the API itself.
Could you take a look at https://github.com/bancodobrasil/stop-analyzing-api/issues/27#issuecomment-767501262 to see if you are able to run the referenced code (https://github.com/KarineValenca/stop-analyzing-enhanced-json)?

**Comment by alokprince** [comment]
what can I do to contribute

**Comment by tiagostutz** [comment]
> what can I do to contribute

Hi there @alokprince ! 👋🏻 
Welcome to stop analyzing.!

Let's get something for you to tackle. Have you read the README instructions? Were you able to bring the project up and running? Need some help with that?
That's the first step, to put the project running, ok?

**Comment by ora-zaq** [comment]
Hi, I'm a newbie to contribute OSS project. (I have a little experience with golang )
I'll prepare to tackle some issues.
